.vscode/settings.json

```diff
@@ -8,13 +8,7 @@
     // already have a Prettier plugin, so don't need this.
```

Comment:
Comments are not allowed in .json

Reply:
No, now it's the age of JSON5 and we are using it. This is not a problem.

Reply:
This does not work in every env though. But yeah, works in VSCode.